source/declarators.tex

```diff
@@ -2110,7 +2110,7 @@
     attribute-specifier-seq\opt decl-specifier-seq\opt declarator virt-specifier-seq\opt{} \terminal{ = default ;}
 \end{ncbnf}
 
-is called an \grammarterm{explicitly-defaulted} definition.
+is called an \term{explicitly-defaulted} definition.
 A function that is explicitly defaulted shall
```

Comment:
In my opinion, \term should only (if at all) survive for stuff that is local enough not to ever be referenced from elsewhere, and thus not needing an index entry at all. For this particular case, it seems the \indextext from before should simply be folded into the \defnx to be used after "is called an".

I'm slightly concerned that we seem to define "explicitly-defaulted definition" as the technical term, although we mean "explicitly-defaulted function definition". That probably needs a bit of research what we actually want here.